distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -41,3 +41,26 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
     // permission java.io.FilePermission "/test/some/monitored/directory", "read";
     // permission java.io.FilePermission "/test/some/monitored/directory${/}-", "read, write";
 }
+
+//                        URL Resource Reader Permissions
+// Adding required permissions for the URL Resource Reader to retrieve file resources.
+//
+//
+// Configuring the URL Resource Reader requires adding read permissions to the directory being read from.
+// For the following 2 permissions are required:
+//   1. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+//   2. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read";
+// Replacing <DIRECTORY_PATH> with the path of the directory containing file resources.
+//
+// Line 1 gives the URL Resource Reader the permissions to read from the directory. Line 2 gives the
+// Resource Reader the permissions to recursively read from the directory path, specified
+// by the directory path's suffix "${/}-".
+//
+//
```

Comment:
✏️ Nit: Remove extra empty `//` line to be consistent with CDM block above.

Reply:
updated